demisto_sdk/commands/common/tests/field_validator_test.py

```diff
@@ -475,59 +475,337 @@ def test_is_valid_from_version_field(
+                    "fromVersion": "5.0.0",
+                    "content": True,
+                    "group": INDICATOR_GROUP_NUMBER,
+                    "selectValues": ["", "option 1"],
+                },
+                False,
+            ),
+            (
+                {
+                    "type": "multiSelect",
+                    "cliName": "test_indicator",
+                    "version": -1,
+                    "fromVersion": "5.0.0",
+                    "content": True,
+                    "group": INDICATOR_GROUP_NUMBER,
+                    "selectValues": [],
+                },
+                True,
+            ),
+            (
+                {
+                    "type": "multiSelect",
+                    "cliName": "test_indicator",
+                    "version": -1,
+                    "fromVersion": "5.0.0",
+                    "content": True,
+                    "group": INDICATOR_GROUP_NUMBER,
+                    "selectValues": None,
+                },
+                True,
+            ),
+        ],
+    )
+    def test_validate_no_empty_selected_values_value_indicator(
+        self, pack, indicator_content, expected_results
+    ):
         """
         Given
-        - An indicator field.
+        - An indicator field content and expected results.
+        - Case 1: singleSelect type indicator field with only empty value in the selectValues field.
+        - Case 2: singleSelect type indicator field with two empty values and a none-empty variable in the selectValues field.
+        - Case 3: singleSelect type indicator field with one empty value and a none-empty variable in the selectValues field.
+        - Case 4: singleSelect type indicator field with an empty list in the selectValues field.
+        - Case 5: singleSelect type indicator field with selectValues field set to None.
+        - Case 6: multiSelect type indicator field with only empty value in the selectValues field.
+        - Case 7: multiSelect type indicator field with two none-empty variables in the selectValues field.
+        - Case 8: multiSelect type indicator field with one empty value and a none-empty variable in the selectValues field.
+        - Case 9: multiSelect type indicator field with an empty list in the selectValues field.
+        - Case 10: multiSelect type indicator field with selectValues field set to None.
 
         When
         - Validating its selectValues do no contain empty values.
 
         Then
-        - Ensure false is returned.
+        - Ensure the right result is returned.
+        - Case 1: Should return False.
+        - Case 2: Should return False.
+        - Case 3: Should return True.
+        - Case 4: Should return True.
+        - Case 5: Should return True.
+        - Case 6: Should return False.
+        - Case 7: Should return True.
+        - Case 8: Should return False.
+        - Case 9: Should return True.
+        - Case 10: Should return True.
```

Comment:
```suggestion
```
This is already written in the test args

Reply:
WDYM?